src/index.js

```diff
@@ -1,16 +1,28 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
+const Title = () => {
+  useEffect(() => {
+    document.title = 'Calculator';
+  });
+  return (
+    <section hidden><div>Lets do some Math</div></section>
+  );
+};
+
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Title />
+    <Calculator />
   </React.StrictMode>,
 );
```

Comment:
- [ ] Great job working on the UI design but I think here you should just keep the app file iside your `index.js` and keep it and you will work with your `app.js` file and you will import all your components there and just the index.js has to compile `app.js` that's the struture of React and it is very important to follow this flow 😊💯💪🏻🚀 so kindly make a new file called app.js and import your `Calculator.js` inside it. ☺💪🏻💯💯💯



![image](https://user-images.githubusercontent.com/58553711/197653665-d039bbd2-5a4c-4e79-9af0-ea976b813021.png)